generics/services/survey.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * name : projects.js
```

Comment:
@borkarsaish65 correct the file description

Reply:
done